dkc/core/tasks.py

```diff
@@ -1,10 +1,15 @@
 from celery import shared_task
 
-from dkc.core.models import File
+from dkc.core.models import File, Folder
 
 
 @shared_task()
 def file_compute_sha512(file_id: int):
     file = File.objects.get(pk=file_id)
     file.compute_sha512()
     file.save()
+
+
+@shared_task()
+def delete_folder(folder_id: int):
+    Folder.objects.get(pk=folder_id).delete()
```

Comment:
Do we need any error handling here if this is queued twice?  Given how this will look on the client side, it doesn't seem like a totally unexpected thing to occur.

Reply:
If I just say "no", is that acceptable?

Reply:
I guess.  I'm just imagining the sentry alerts getting raised because of it.

Reply:
I can see this happening if a user deletes a subfolder, then immediately goes up and deletes its parent folder. Otherwise I wouldn't anticipate much noise.

I'd like to punt on this and address it if and when we hear Sentry complaining.